cmd/fetcher/main.go

```diff
@@ -48,18 +46,22 @@ func run(ctx context.Context, root string, depth int) error {
 		return err
 	}
 	client := fetchclient.New()
+	latestVersions := make(map[string]string, len(configs))
 	for _, config := range configs {
 		if config.Source.Disabled {
 			log.Printf("skipping source: %s\n", config.Filename)
 			continue
 		}
-		newVersion, err := client.Fetch(ctx, config)
-		if err != nil {
-			return err
+		newVersion := latestVersions[config.CacheKey()]
+		if newVersion == "" {
+			newVersion, err = client.Fetch(ctx, config)
+			if err != nil {
+				return err
+			}
+			latestVersions[config.CacheKey()] = newVersion
 		}
 		// For now we ignore prerelease versions. But this may change in the future.
 		if semver.Prerelease(newVersion) != "" && !config.IncludePrerelease {
-			log.Printf("%s: skipping prerelease version: %s", config.Source.Name(), newVersion)
```

Comment:
Removed a bunch of extra logging so Github actions log will only show the new discovered plugins.